src/cascadia/TerminalApp/AppCommandlineArgs.h

```diff
@@ -106,7 +106,7 @@ class TerminalApp::AppCommandlineArgs final
     std::string _exitMessage;
     bool _shouldExitEarly{ false };
 
-    std::optional<int> _windowTarget{ std::nullopt };
+    std::string _windowTarget{ "" };

```

Comment:
```suggestion
    std::string _windowTarget{};
```

same reason as for hstring :smile: